lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -181,3 +236,45 @@ instance Accept Any where
         [ "application" // "json"
         , "application" // "json" /: ("charset", "utf-8")
         ]
+
+{-------------------------------------------------------------------------------
+                               Api Layer
+-------------------------------------------------------------------------------}
+
+data ApiLayer s t (k :: Depth -> * -> *)
```

Comment:
What is this? What is the difference between it and the `WalletLayer` now?

Reply:
The `ApiLayer` does not have any handle on a database, instead, it has a Worker registry (whereas the `WalletLayer` is foreign to the registry, but has a handle on a db connection). We use the `ApiLayer` in.. the Api, and create a `WalletLayer` out of it in every handlers (if such a wallet is registered).